.github/workflows/lint.yaml

```diff
@@ -50,3 +50,20 @@ jobs:
         run: python -m pip install gitlint
       - name: Run gitlint check
         run: gitlint --commits origin/${{ github.event.pull_request.base.ref }}..HEAD
+  checkton:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Repository
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
```

Comment:
In the original [PR](https://github.com/konflux-ci/build-definitions/pull/1051/files) this has a comment "Differential Checkton requires full git history" - but is that really true? Shouldn't one level be enough for diff?

Reply:
I have tested it with 'one level ', and it works. I don't mind removing it; I'm just following the [docs for the action.](https://github.com/chmeliik/checkton) What do you think is best? 👍🏻

Reply:
Oh, now I realize `fetch-depth: 0` may actually be really needed - that's because a PR can contain multiple commits and so with just one level deep you might not be able to do the diff. So yeah, let's keep this.